app/registries/foreman/plugin/rbac_support.rb

```diff
@@ -4,8 +4,10 @@ class RbacSupport
       def add_all_permissions_to_default_roles(all_permissions)
         return if Foreman.in_rake?('db:migrate')
         view_permissions = all_permissions.where("name LIKE :name", :name => "view_%")
+        org_admin_permissions = all_permissions.where("resource_type <> :resource_type", :resource_type => "Organization")
```

Comment:
I agree, but it turns out that this would ignore permissions that has NULL as resource_type (generic permissions), so I needed to add "OR" to the condition, AR in Rails 4 don't have DSL for it and I'd like to avoid using AREL directly.